app/src/sizes.sass

```diff
@@ -1,5 +1,5 @@
 $contmaxwidth: 960px
-$navbarheight: 40px
+$navbarheight: 45px
```

Comment:
Why did you change that? Moreover, when that height is changed you must also change the margins of h1 and h2 to keep them vertically centered.